docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -4785,7 +4785,6 @@ image](#create-an-image) section for more details.
     Content-Type: application/json
 
     {
-      "Name": "top",
```

Comment:
And, yes, I was wondering if there should be a separate `ServiceUpdate` API type (without the "Name"). IIUC, the overall design is that `Update` request should only have to send those parts of the Service / Spec that need to be _modified_ (only a "diff"), correct?